src/components/templates/flex/index.stories.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react"
+import { storiesOf } from "@storybook/react"
+import { readmeCleanup } from "utils/readme"
+import { H4, Text, TextSmall } from "src/components/typography"
+// @ts-ignore
+import readme from "./README.md"
+
+import Flex from "./index"
+
+const flexStory = storiesOf("COMPONENTS|Templates/Flex", module)
+
+const subData = {
+  readme: {
+    sidebar: readmeCleanup(readme),
+  },
+  jest: ["index.test.js"],
+}
+
+const Card = () => (
+  <Flex
+    height="260px"
+    width={{ max: "220px" }}
+    background="disabled"
+    padding={[4]}
```

Comment:
I suppose this is pixels so this defeats the 8px (or 4px) breakpoint system we discussed. But I guess that would be ok for now.

Reply:
padding, margin and gap are multipliers of `SIZE_SUB_UNIT` = 4px. 
So, the `padding={[4]}` evaluates to `padding: 16px;`